assets/scss/templates/_signup.scss

```diff
@@ -175,9 +181,3 @@
     }
   }
 }
-
-.link--title {
-  font-weight: $dp-font-weight-bold;
-  text-transform: uppercase;
-  font-style: normal;
-}
```

Comment:
check on the webapp

Reply:
But first is modified in the library and after is modified in the webapp ?

Reply:
Right, you can test it in webapp before merging, let me know if you need help on that.